src/sharedwithyou.cs

```diff
@@ -0,0 +1,398 @@
+	[Native]
+	public enum SWAttributionViewHorizontalAlignment : long {
+		Default = 0,
+		Leading,
+		Center,
+		Trailing,
+	}
+
+	[TV (16,0), NoMac, iOS (16,0)]
+	[Native]
+	public enum SWHighlightCenterErrorCode : long {
+		NoError = 0,
+		InternalError,
+		InvalidURL,
+		AccessDenied,
+	}
+
+	[NoTV, NoWatch, NoMac, iOS (16,0), MacCatalyst (16,0)]
+	[Native]
+	public enum SWHighlightChangeEventTrigger : long {
+		Edit = 1,
+		Comment = 2,
+	}
+
+	[NoTV, NoWatch, NoMac, iOS (16,0), MacCatalyst (16,0)]
+	[Native]
+	public enum SWHighlightMembershipEventTrigger : long {
+		AddedCollaborator = 1,
+		RemovedCollaborator = 2,
+	}
+
+	[NoTV, NoWatch, NoMac, iOS (16,0), MacCatalyst (16,0)]
+	[Native]
+	public enum SWHighlightPersistenceEventTrigger : long {
+		Created = 1,
+		Deleted = 2,
+		Renamed = 3,
+		Moved = 4,
+	}
+
+	[NoWatch, NoMac, TV (16,0), iOS (16,0), MacCatalyst (16,0)]
+	[BaseType (typeof (UIView))]
+	interface SWAttributionView
+	{
+
+		[DesignatedInitializer]
+		[Export ("initWithFrame:")]
+		NativeHandle Constructor (CGRect frame);
+
+		[NullAllowed, Export ("highlight", ArgumentSemantic.Strong)]
+		SWHighlight Highlight { get; set; }
+
+		[Export ("displayContext", ArgumentSemantic.Assign)]
+		SWAttributionViewDisplayContext DisplayContext { get; set; }
+
+		[Export ("horizontalAlignment", ArgumentSemantic.Assign)]
+		SWAttributionViewHorizontalAlignment HorizontalAlignment { get; set; }
+
+		[Export ("backgroundStyle", ArgumentSemantic.Assign)]
+		SWAttributionViewBackgroundStyle BackgroundStyle { get; set; }
+
+		[Export ("preferredMaxLayoutWidth")]
+		nfloat PreferredMaxLayoutWidth { get; set; }
+
+		[Export ("highlightMenu", ArgumentSemantic.Strong)]
+		UIMenu HighlightMenu { get; }
+
+		[NullAllowed, Export ("menuTitleForHideAction", ArgumentSemantic.Strong)]
+		string MenuTitleForHideAction { get; set; }
+
+		[NullAllowed, Export ("supplementalMenu", ArgumentSemantic.Strong)]
+		UIMenu SupplementalMenu { get; set; }
+
+		[NoMac, NoiOS]
+		[Export ("enablesMarquee")]
+		bool EnablesMarquee { get; set; }
+	}
+
+	[NoWatch, TV (16,0), NoMac, iOS (16,0), MacCatalyst (16,0)]
+	[BaseType (typeof (NSObject))]
+	[DisableDefaultCtor]
+	interface SWHighlight : NSSecureCoding, NSCopying
+	{
+
+		[NoTV]
+		[Field ("SWCollaborationMetadataTypeIdentifier")]
+		NSString MetadataTypeIdentifier { get; }
+
+		[Export ("identifier", ArgumentSemantic.Copy)]
+		NSObject/*<NSSecureCoding, NSCopying>*/ Identifier { get; }
+
+		[Export ("URL", ArgumentSemantic.Copy)]
+		NSUrl Url { get; }
+	}
+
+	interface ISWHighlightCenterDelegate {}
+
+	[NoWatch, TV (16,0), NoMac, iOS (16,0), MacCatalyst (16,0)]
+#if NET
+	[Protocol, Model]
+#else
+	[Protocol, Model (AutoGeneratedName = true)]
+#endif
+	[BaseType (typeof (NSObject))]
+	interface SWHighlightCenterDelegate
+	{
+		[Abstract]
+		[Export ("highlightCenterHighlightsDidChange:")]
+		void HighlightCenterHighlightsDidChange (SWHighlightCenter highlightCenter);
```

Comment:
```suggestion
		void DidHighlightCenterHighlightsChange (SWHighlightCenter highlightCenter);
```
perhaps verbify - not sure if this changes a pattern

Reply:
I think it is better to do
```suggestion
		void HighlightsDidChange (SWHighlightCenter highlightCenter);
```